static/js/admin/src/ResultView.js

```diff
@@ -242,6 +242,21 @@ define('ResultView', [ 'jquery', 'jquery-ui', 'ResultObjectList', 'URLUtilities'
 			this.resizeResults();
 			this.$window.resize($.proxy(this.resizeResults, this));
 		},
+
+		// Set count of descendants
+		setChildCounts: function(container) {
```

Comment:
this is clever, but seems like a lot of work. I think you could change the response from the controller to add a count in for the parent container.
In here:
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/fcrepo4/admin/src/main/java/edu/unc/lib/dl/admin/controller/SearchController.java#L173

Pass `resultResponse.getSelectedContainer()` into `childrenCountService.addChildrenCounts` similar to https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/9cad1419fa96a243348f2a9e8307aa0b4be44079/admin/src/main/java/edu/unc/lib/dl/admin/controller/AbstractSearchController.java#L63

You would probably have to wrap it in a temporary list